libs/application/templates/aosh/work-accident-notification/src/fields/CauseOfInjury/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { FieldBaseProps } from '@island.is/application/types'
```

Comment:
I checked out the application and looked at this component. I think it's possible to do this with shared components from the application system. In `buildSelectField` it can take in the option `isMulti: true`

https://ui.devland.is/something?path=/docs/application-system-selectformfield--docs

And the radio field at the bottom can take in dynamic options based on the answers object.

Reply:
This also goes for all the other custom components using `<CausesAndEffect />`

Reply:
Would be great if thats possible! but is it acceptable to push this as is to get VER to test the main features and functionality of the application on DEV and I will look into if this is possible before next push ? I don't see this being used anywhere and we made some additions to the Select component so I can't tell for certain that will work for us.

Reply:
This component we’ve implemented isn’t not ready, but rather matches the behaviors we need. If it’s possible to achieve the same functionality with buildSelectField and buildRadio, we’d prefer that approach, but we’re not sure it’s currently possible. For instance, we’re using properties like withCheckmark (new), isSelected (new), and hideSelectedOptions, which aren’t available in buildSelectField at the moment.
Since I haven’t modified or added to these core application components before, I’m unsure of the effort involved and the potential impact on other components, especially considering shared types like Option. We’ve received approval from Norda for this different Select design.
Regarding the deploy-feature flag, I have limited experience with it, but my team members mentioned it takes some effort to set up so it works properly. However, since this application is fairly isolated and under the feature flag, it shouldn’t cause issues on main regardless of this specific components readiness or if changes/improvements are needed.

Reply:
`deploy-feature` shouldn't require any setup, you just add the tag on the PR and the github-actions bot posts links on the PR. The links should work shortly after all checks have been passed

![image](https://github.com/user-attachments/assets/e442439b-f510-4dea-91f6-37a57977c928)

Reply:
Could you respond to the first two topics of the comment above ?

Reply:
Yes, If it works with shared components, then that is most preferable. But if not then it has to be custom.
If there is something that can't be tested with `deploy-feature`, then you can merge the branch onto main behind a feature-flag.